debug/src/debug.js

```diff
@@ -214,8 +214,8 @@ export function initDebug() {
 			checkPropTypes(
 				vnode.type.propTypes,
 				vnode.props,
-				getDisplayName(vnode),
-				serializeVNode(vnode)
+				'prop',
```

Comment:
TIL that "prop types" can also be used to check things that are not props, such as context. See https://github.com/facebook/prop-types/blob/1c4077b7455e037bd8f81f48e9c51d60c972f8e9/checkPropTypes.js#L37.